integration/tests/conftest.py

```diff
@@ -508,3 +495,59 @@ def neon_price(web3_client_session) -> float:
     price = web3_client_session.get_token_usd_gas_price()
     with allure.step(f"NEON price {price}$"):
         return price
+
+
+@pytest.fixture(scope="class")
+def events_checker_contract(web3_client, accounts) -> tp.Any:
+    contract, _ = web3_client.deploy_and_get_contract("common/tracer/ContractCaller", "0.8.15", account=accounts[0])
+    yield contract
+
+
+@pytest.fixture(scope="class")
+def counter_contract(web3_client, accounts):
+    contract, _ = web3_client.deploy_and_get_contract("common/Counter", "0.8.10", account=accounts[0])
+    return contract
+
+
+@pytest.fixture(scope="class")
+def nested_call_contracts(accounts, web3_client):
+    contract_a, _ = web3_client.deploy_and_get_contract(
+        "common/NestedCallsChecker", "0.8.12", accounts[0], contract_name="A"
+    )
+    contract_b, _ = web3_client.deploy_and_get_contract(
+        "common/NestedCallsChecker", "0.8.12", accounts[0], contract_name="B"
+    )
+    contract_c, _ = web3_client.deploy_and_get_contract(
+        "common/NestedCallsChecker", "0.8.12", accounts[0], contract_name="C"
+    )
+    yield contract_a, contract_b, contract_c
+
+
+@pytest.fixture(scope="function")
+def recursion_factory(accounts, web3_client):
+    sender_account = accounts[0]
+    contract, _ = web3_client.deploy_and_get_contract(
+        "common/Recursion",
+        "0.8.10",
+        sender_account,
+        contract_name="DeployRecursionFactory",
+        constructor_args=[3],
+    )
+    yield contract
+
+
+@pytest.fixture(scope="function")
+def destroyable_contract(accounts, web3_client):
```

Comment:
please delete it from TestSelfDestructOpcode class than you moved it

Reply:
the same question about all new fixtures

Reply:
Fixed ✅